docs/core/geometry.md

```diff
@@ -7,187 +7,211 @@ of higher complexity. This design has been chosen as the surface objects can be
-| `PerigeeSurface`, `StrawSurface` | [d, z] | `CylinderBounds`                                                                            |
-
-![CylinderBounds](/figures/geometry/CylinderBounds.png)
-![DiscBounds](/figures/geometry/DiscBounds.png)
-![PlaneBounds](/figures/geometry/PlaneBounds.png)
+| {class}`Acts::ConeSurface`         | $[r\phi, z]$         | {class}`Acts::ConeBounds`                                                                                |
+| {class}`Acts::CylinderSurface`     | $[r, \phi]$         | {class}`Acts::CylinderBounds`                                                                            |
+| {class}`Acts::DiscSurface`         | $[r, \phi]$         | {class}`Acts::RadialBounds`, {class}`Acts::DiscTrapezoidalBounds`                                                     |
+| {class}`Acts::PlaneSurface`        | $[x, y]$         | {class}`Acts::RectangleBounds`, {class}`Acts::TrapezoidalBounds`, <br>{class}`Acts::TriangleBounds`,{class}`Acts::InfiniteBounds`, <br> {class}`Acts::EllipseBounds` |
+| {class}`Acts::PerigeeSurface`,<br> {class}`Acts::StrawSurface` | $[d, z]$ | {class}`Acts::CylinderBounds`                                                                            |
+| {class}`Acts::LineSurface` | $[d_0, z_0]$ | {class}`Acts::LineBounds` |
 
 ```{tip}
-The coordinate systems define - in an ideal setup - also the readout measurement directions. In such a case, a track prediction from the propagation, e.g. will already be in the correct frame of the measurement and residual or compatibility checks will not need additional coordinate transformations.
+The coordinate systems define - in an ideal setup - also the readout
+measurement directions. In such a case, a track prediction from the
+propagation, e.g. will already be in the correct frame of the measurement and
+residual or compatibility checks will not need additional coordinate
+transformations.
 ```
 
+### Plane surface
+
+![PlaneBounds](/figures/geometry/PlaneBounds.png)
+
+:::{doxygenclass} Acts::PlaneSurface
+---
+members: globalToLocal,localToGlobal,intersect,normal
+---
+:::
+
+### Disc surface
+
+![DiscBounds](/figures/geometry/DiscBounds.png)
+
+:::{doxygenclass} Acts::DiscSurface
+---
+members: globalToLocal,localToGlobal,intersect,normal
+---
+:::
+ 
+### Cylinder surface
+
+![CylinderBounds](/figures/geometry/CylinderBounds.png)
+
+:::{doxygenclass} Acts::CylinderSurface
+---
+members: globalToLocal,localToGlobal,intersect,normal
+---
+:::
+
+### Cone surface
+
+:::{doxygenclass} Acts::ConeSurface
+---
+members: globalToLocal,localToGlobal,intersect,normal
+---
+:::
+
+### Line surface
+
+{class}`Acts::LineSurface` is a special kind of surface that depends on a reference
+direction, typically the unit momentum direction of a particle. A point in
+space is considered *on surface* if and only if it coincides with the point of
+closest approach between the direction vector $\vec d$ and the line direction
```

Comment:
I don't understand what the direction vector is, could you please explain?

Reply:
It's just the momentum direction vector that is mentioned earlier in the paragraph.

Reply:
I would then write 
`... typically the unit momentum direction $\vec d$ of a particle.`
to make things even clearer